sdks/python/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "hatchet-sdk"
-version = "1.7.0"
+version = "1.8.0"
```

Comment:
minor or patch?

Reply:
🤷 I was thinking minor b/c it adds a couple new features (expanded task defaults) but either way is fine by me

Reply:
your call